apps/explorer/src/app/components/jump-to-block/index.tsx

```diff
@@ -20,8 +20,20 @@ export const JumpToBlock = () => {
 
   return (
     <form onSubmit={handleSubmit}>
-      <input type={'tel'} name={'blockNumber'} placeholder={'Block number'} />
-      <input type={'submit'} value={'Go'} />
+      <label
+        htmlFor="block-input"
+        className="block uppercase text-h5 font-bold"
+      >
+        Jump to block
+      </label>
+      <input
```

Comment:
You could use the input component from ui toolkit now? 

You'll need to either:
- Update the input to allow rendering of a integrated button. 
- Or (and IMO the simpler/better solution for now) just use the input and button components from ui toolkit wrapped in a container with display flex.

Reply:
Will do. I'm going to get this branch sorted without it for now as I want to unblock Edd, but I will revisit this. https://github.com/vegaprotocol/frontend-monorepo/issues/76